tools/build_variables.py

```diff
@@ -369,6 +369,7 @@ def add_torch_libs():
         deps=[
             ":libtorch",
             ":thnn",
+            "//caffe2/torch/fb/init:init",
```

Comment:
just fyi, the hypothetical bazel user of open-source doesn't have torch/fb namespace.
There isn't such a user who exists, and they can filter things out if they want with a regex, but just fyi

Reply:
hm, I don't think we added build_variables.py for a hypothetical bazel user. It's just to have a commonly-changing file list in github repo. For one thing, it's already full with buck-specific directives like external_deps.

I'd love to get some time and refactor this stuff to have a dedicated (plain test?) file with lists of sources that gets added in both CMake and BUCK. That would be ideal.